src/php/README.md

```diff
@@ -365,4 +365,30 @@ $client = new Helloworld\GreeterClient('localhost:50051', [
 ]);
 ```
 
-[Node]:https://github.com/grpc/grpc/tree/master/src/node/examples
+### Compression 
+
+You can customize the compression behavior on client side, specify this `grpc.default_compression_algorithm (int)` option control compression algorithm for the channel and `grpc.default_compression_level (int)` option control  compression level for the channel.
```

Comment:
"You can customize the compression behavior on the client side, by specifying the following options when constructing your PHP client."